### DISCUSSION TITLE
Can I port the hashing algorithm to D?

### DISCUSSION THREAD

**Discussion by ZILtoid1991** [tkaitchuck/aHash#212] [Q&A] [answered]
If yes, do I have to transfer the MIT/Apache license to my project?

**Comment by tkaitchuck** [comment, marked as answer]
I would recommend starting with just the fallback algorithm using folded multiply. This is the most general.
The licences apply to all the code in the repo. But take a look at the licences for the specific terms, they are fairly permissive.
